examples/agent/query_openai_agent.ts

```diff
@@ -33,12 +33,14 @@ async function main() {
   });
 
   // Chat with the agent
-  const response = await agent.chat({
+  const {
+    message: { content },
```

Comment:
won't it be easier to read to return `message` and log `message.content`?
```suggestion
    message,
```

Reply:
Hi @marcusschiesser thanks for reviewing my MR! I think your code snippet is easier to read, I've changed to follow your code, thank you~